accepted/future-releases/static-extension-methods/feature-specification.md

```diff
@@ -554,7 +566,7 @@ Since it's possible to add extensions on superclass (including `Object`), it wou
 
 - Otherwise, the single most-specific extension's member is invoked with the extension's type parameters bound to the types found by inference, and with `this ` bound to the receiver.
 
-- An extension method can be invoked explicitly using the syntax `ExtensionName(object).method(args)`. Type arguments can be applied to the extension explicitly as well, `MyList<String>(listOfString).quickSort()`. Such an invocation overrides all extension resolution. It is a compile-time error if `ExtensionName` would not apply to the `object.method(args)` invocation if it was in scope. 
+- An extension method can be invoked explicitly using the syntax `ExtensionName(object).method(args)`. Type arguments can be applied to the extension explicitly as well, `MyList<String>(listOfString).quickSort()`. Such an invocation overrides all extension resolution. It is a compile-time error if `ExtensionName` would not apply to the `object.method(args)` invocation if it was in scope. It is a compile-time error if the extension `ExtensionName` doesn't declare a member named `method`.
```

Comment:
I don't know what the sentence starting with "It is a compile-time error if `ExtensionName`" is supposed to mean, consider rephrasing?

Also, it's not clear to me where we specify that `ExtensionName(object)` is an error if the resolved `on` type for `ExtensionName` is not assignable to the static type of `object`, which I presume we want?